locales/locales.ini

```diff
@@ -20,3 +20,6 @@
 
 [nl]
 @import url(notes.nl.properties)
+
+[uk]
+@import url (notes.uk.properties)
```

Comment:
Oops, already fixed.
Ok, I'll test it and inform about the result here.
Thanks!